common/screenshotter.go

```diff
@@ -121,12 +121,14 @@ func (s *screenshotter) originalViewportSize(p *Page) (*Size, *Size, error) {
 	if err != nil {
 		return nil, nil, fmt.Errorf("getting viewport dimensions: %w", err)
 	}
-	r, ok := result.(goja.Object)
-	if !ok {
-		return nil, nil, fmt.Errorf("cannot convert to goja object: %w", err)
+
+	var returnVal Size
+	if err := convert(result, &returnVal); err != nil {
+		k6ext.Panic(s.ctx, "unpacking selected options: %w", err)
```

Comment:
Return an error?

Reply:
🤦 Yep, bad copy and paste.

Reply:
Resolved in 74c783c4448cdaeb00523b75b1e2e162f10c6188